ppr-ui/src/store/store.ts

```diff
@@ -180,6 +208,13 @@ export const useStore = defineStore('assetsStore', () => {
       useMhrValidations(modelRef).getStepValidation(MhrSectVal.HOME_OWNERS_VALID) &&
       useMhrValidations(modelRef).getStepValidation(MhrSectVal.LOCATION_VALID)
   })
+  const isMhrManufactuerRegistrationReviewValid = computed<boolean>(() => {
+    const modelRef = toRefs(getMhrRegistrationValidationModel.value)
+    return state.value.mhrValidationManufactuerState.reviewConfirmValid.authorizationValid &&
+      state.value.mhrValidationManufactuerState.reviewConfirmValid?.attentionValid &&
+      state.value.mhrValidationManufactuerState.reviewConfirmValid?.folioOrRefNumValid &&
+      useMhrValidations(modelRef).getStepValidation(MhrSectVal.YOUR_HOME_VALID)
+  })
```

Comment:
Wouldn't the `getStepValidation` function serve the same purpose? 
This does appear to consider the filing as a whole as well as review.

Reply:
Yeah I think it does, I mostly copied this section from what was already there. So was not sure if there is something I might be missing.

Reply:
Ah i see. Fair enough!
Yes, i'm sure it has it's purpose, although both validate steps and app would be true at the point of considering if the filing as a whole was good to go. 
It MAY work with the common getSteps function, worth a try but no action required 👍